LICENSE

```diff
@@ -269,13 +284,19 @@ http://github.com/necolas/normalize.css
 Copyright (c) Nicolas Gallagher and Jonathan Neal
 License: https://github.com/necolas/normalize.css/blob/master/LICENSE.md
 
+FIXME: I cannot find this source code in our Git repository. am I correct in
+thinking this is no longer used or is a build time inclusion?
+
```

Comment:
@mxm

Reply:
Not sure, maybe @m-kharbat knows about this one?